discovery-provider/src/queries/health_check.py

```diff
@@ -131,3 +132,10 @@ def ipld_block_check():
 def ip_check():
     ip = helpers.get_ip(request)
     return success_response(ip, sign_response=False)
+
+
+@bp.route("/db_seed_restore_check", methods=["GET"])
```

Comment:
new route so health check / startup time don't get impacted but we still get visibility